Assets/Scripts/Ships/Fleets/FleetManager.cs

```diff
@@ -21,6 +21,8 @@ public class FleetManager : MonoBehaviour
         [ReadOnlyField] [SerializeField] private int fleetId;
         [SerializeField] private GameObject shipBasePrefab;
 
+        private InfoWindow _infoWindow;
+
```

Comment:
Don't think the FleetManager is the right spot for this, since player & enemy fleets will both be using it.

Reply:
Yeah I'll change this for the attack refactor